lib/cfonb/operation_detail/lc2.rb

```diff
@@ -3,8 +3,14 @@
 module CFONB
   module OperationDetail
     class LC2
+      ATTRIBUTES = %i[unstructured_label_2].freeze
+
       def self.apply(operation, line)
-        operation.label += "\n#{line.detail.strip}"
+        operation.unstructured_label_2 = if operation.unstructured_label_2.nil?
+          line.detail.strip.to_s
+        else
+          "#{operation.unstructured_label_2}\n#{line.detail.strip}"
+        end
```

Comment:
I think there can only be one line of this
```suggestion
        operation.unstructured_label_2 = line.detail.strip
```